cmd/argo/commands/wait.go

```diff
@@ -0,0 +1,111 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	wfclient "github.com/argoproj/argo/workflow/client"
+	goversion "github.com/hashicorp/go-version"
+	"github.com/jpillora/backoff"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(waitCmd)
+	waitCmd.Flags().StringVarP(&waitArgs.namespace, "namespace", "n", "default", "Namespace in which to watch workflows")
```

Comment:
The namespace flag is already part of the global kubernetes flags that we add to our commands, and should not be necessary to add to this specific command.

Reply:
Removed.